config/spring19_seminar/cola_var/cola_bilstm.conf

```diff
@@ -0,0 +1,31 @@
+// Import the defaults using the relative path
+include "../final.conf"
+
+
+// Output path
+project_dir = ${JIANT_PROJECT_PREFIX}
+
+
+// Optimization
+lr = 3e-4  // following "cola_elmo.conf"
+
+
+// Target tasks
+do_target_task_training = 1  // If true, after do_pretrain train the task-specific model parameters
+write_preds = "val,test"  // 0 for none, or comma-separated splits in {"train", "val", "test"} 
+                          // for which predictions are written to disk during do_full_eval
+
+
+// Pretraining tasks
+load_model = 0  // If true, restore from checkpoint when starting do_pretrain
+
+
+// Model
+// Model, biLSTM
```

Comment:
Probably simplest to use the same batch size for both the BiLSTM and BERT.

Reply:
fixed